src/components/touchables/TouchableOpacity.tsx

```diff
@@ -38,9 +39,9 @@ export default class TouchableOpacity extends Component {
     }).start();
   };
 
-  onStateChange = (from, to) => {
+  onStateChange = (_: number, to: number) => {
     if (to === TOUCHABLE_STATE.BEGAN) {
-      this.setOpacityTo(this.props.activeOpacity, 0);
+      this.setOpacityTo(this.props.activeOpacity!, 0);
```

Comment:
Not null type assertion.

Reply:
It is inevitable in class component with defaultProps, which take care of always providing the value.